diff -urN a/livecd.py b/livecd.py
--- a/livecd.py	2015-03-24 11:18:46.000000000 -0400
+++ b/livecd.py	2016-01-28 19:30:09.000000000 -0500
@@ -341,9 +341,10 @@
 
         # now write out the "real" fstab and mtab
         anaconda.id.storage.write(anaconda.rootPath)
-        f = open(anaconda.rootPath + "/etc/mtab", "w+")
-        f.write(anaconda.id.storage.mtab)
-        f.close()        
+        if not os.path.islink(anaconda.rootPath + "/etc/mtab"):
+            f = open(anaconda.rootPath + "/etc/mtab", "w+")
+            f.write(anaconda.id.storage.mtab)
+            f.close()
         
         # copy over the modprobe.conf
         if os.path.exists("/etc/modprobe.conf"):
