From c0b54f28825490d161550bf6ee47cdf79fa6b45e Mon Sep 17 00:00:00 2001
From: Evgueni Souleimanov <esoule@100500.ca>
Date: Wed, 9 Nov 2016 21:33:41 -0500
Subject: [PATCH] t/fixtures.t: stop using subtests on Test::More 0.92

---
 t/fixtures.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/fixtures.t b/t/fixtures.t
index fda489b..fc13495 100644
--- a/t/fixtures.t
+++ b/t/fixtures.t
@@ -49,11 +49,11 @@ my %STAT = (
 File::StripNondeterminism::init();
 
 $File::StripNondeterminism::canonical_time = 1423159771;
 
 my @fixtures = glob('t/fixtures/*/*.in');
-plan tests => scalar @fixtures;
+plan tests => (scalar @fixtures) * ((scalar keys %STAT) + 3);
 
 foreach my $filename (@fixtures) {
 	# Use a temporary directory per fixture so we can check whether any
 	# extraneous files are leftover.
 	my $temp = tempdir( CLEANUP => 1 );
@@ -63,11 +63,11 @@ foreach my $filename (@fixtures) {
 
 	copy($filename, $in) or die "Copy failed: $!";
 
 	my $normalizer = File::StripNondeterminism::get_normalizer_for_file($in);
 
-	subtest $filename => sub {
+	if (1) {
 		isnt(undef, $normalizer, "Normalizer found for $in");
 
 		my @stat_before = lstat $in;
 		$normalizer->($in) if defined $normalizer;
 		my @stat_after = lstat $in;
@@ -80,10 +80,9 @@ foreach my $filename (@fixtures) {
 		}
 
 		my @files = glob("$temp/*");
 		ok(scalar(@files) == 1, "Unexpected files leftover: " . join(" ", @files));
 
-		done_testing;
 	}
 }
 
 done_testing;
-- 
2.8.0

