From 0689710b369a1c506b6609aa38a5fc34514e65c6 Mon Sep 17 00:00:00 2001
From: Evgueni Souleimanov <esoule@100500.ca>
Date: Sun, 13 Nov 2016 18:59:47 -0500
Subject: [PATCH 2/6] t/fixtures.t: stop using subtests on Test::More 0.92 on
 el6

---
 t/fixtures.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/fixtures.t b/t/fixtures.t
index ca5fd9f..128f217 100644
--- a/t/fixtures.t
+++ b/t/fixtures.t
@@ -48,13 +48,13 @@ my %STAT = (
 
 File::StripNondeterminism::init();
 
 $File::StripNondeterminism::canonical_time = 1423159771;
 
 my @fixtures = glob('t/fixtures/*/*.in');
-plan tests => scalar @fixtures;
+plan tests => (scalar @fixtures) * ((scalar keys %STAT) + 3);
 
 foreach my $filename (@fixtures) {
 	# Use a temporary directory per fixture so we can check whether any
 	# extraneous files are leftover.
 	my $temp = tempdir( CLEANUP => 1 );
 
@@ -62,13 +62,13 @@ foreach my $filename (@fixtures) {
 	(my $out = $filename) =~ s/\.in$/.out/;
 
 	copy($filename, $in) or die "Copy failed: $!";
 
 	my $normalizer = File::StripNondeterminism::get_normalizer_for_file($in);
 
-	subtest $filename => sub {
+	if (1) {
 		isnt(undef, $normalizer, "Normalizer found for $in");
 
 		my @stat_before = lstat $in;
 		$normalizer->($in) if defined $normalizer;
 		my @stat_after = lstat $in;
 
@@ -79,11 +79,10 @@ foreach my $filename (@fixtures) {
 			is($stat_before[$i], $stat_after[$i], "$filename: $STAT{$i}");
 		}
 
 		my @files = glob("$temp/*");
 		ok(scalar(@files) == 1, "Unexpected files leftover: " . join(" ", @files));
 
-		done_testing;
 	}
 }
 
 done_testing;
-- 
2.8.0

